app/views/admins/survey_responses/show.html.erb

```diff
@@ -2,9 +2,15 @@
 <% back_link "Back to Responses", admin_study_survey_survey_responses_path(@survey.study, @survey) %>
 
 <div class="flex gap-y-4">
-  <% @survey_response.responses.each do |r| %>
-    <div class="border bg-slate-50 p-4">
-      <%= r.as_json %>
+  <% if @survey_response.present? && @survey_response.responses.present? %>
```

Comment:
`@survey_response` should not ever be nil, so I think this can safely be written out as `@survey_response.responses.any?` — see here for why I recommend using `any?` over `present?` in this case: https://www.reddit.com/r/ruby/comments/5gtnph/benchmarking_present_vs_any_vs_exists/

(note: `present?` is great and I use it all the time! Just not for associations like this 😄)

Reply:
Great!!